jest/jest.config.js

```diff
@@ -0,0 +1,5 @@
+
+module.exports = {
+  "bail": true,
+  "verbose": true
+}
```

Comment:
I'm all in for modern tools, but I'm wondering if we should add a new one at this point.
I guess these tests can be written using the existing Jasmine setup.

It would be great to migrate them all together to a technology that we choose. Otherwise, it might be yet another thing to configure and run, or it might die if other developers don't use it (as we'd have jasmine vs jest choice now). Also, I know I was talking about using Jest, but Kirk made a really good point about Cypress using Mocha. That's a significant argument - we could pretty much limit all the syntax and matches to mocha/chai style assuming that we also use Cypress. Constant switching between two styles within one project can be annoying.